README.md

```diff
@@ -61,15 +61,15 @@ Here's a basic example of what map configuration files look like:
 
 ## Compiling
 
-1. Clone the repo to your computer.
+1. Clone the repo to your computer. `git clone https://github.com/Warzone/TGM`
 
 2. Compile CraftBukkit by downloading and running the BuildTools jar from [here](https://hub.spigotmc.org/jenkins/job/BuildTools). Make sure you include the `-compile craftbukkit` flag when compiling it, otherwise you might receive errors when later compiling the plugin.
 
 3. Make sure you have Maven installed and run `mvn clean install` in the top-level folder to generate the required libraries. You can then download the jar from the `target` folder.
 
 ## Developer Tips
 
-1. We use [Lombok](https://projectlombok.org/). Make sure you have the Lombok plugin installed on your preferred IDE.
+1. We use [Lombok](https://projectlombok.org/). Make sure you have the Lombok plugin installed on your preferred IDE, otherwise you cannot compile.
```

Comment:
This isn't true, you can still compile directly with Maven even without the Lombok plugin installed on your IDE.